mmv1/templates/terraform/decoders/dlp_stored_info_type.go.erb

```diff
@@ -16,4 +16,9 @@ config := configRaw.(map[string]interface{})
 // Name comes back on the top level, so set here
 config["name"] = name
 
+configMeta := meta.(*transport_tpg.Config)
+if err := d.Set("stored_info_type_id", flattenDataLossPreventionStoredInfoTypeName(res["name"], d, configMeta)); err != nil {
```

Comment:
Ah, is this why `default_from_api` is set? So it can be Optional and Computed and we have the ability to set it? It looks here that we'll always be setting it to name though? If the user set's it in the config, this isn't using that value.

Reply:
Actually, the `storedInfoTypeId` field can be sent in the request body of the [POST endpoint](https://cloud.google.com/dlp/docs/reference/rest/v2/projects.locations.storedInfoTypes/create#request-body) but the response from [GET endpoint](https://cloud.google.com/dlp/docs/reference/rest/v2/projects.locations.storedInfoTypes#StoredInfoType) won't contain the `storedInfoTypeId` field; instead, it will be present in the `name` field. Hence, I've used the flatten function of the `name` field.